### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by JasonLewisMSFT**
Updating channels limits with new release to support 1K channels today. With this release we support any combination of standard and shared channels including up to 30 private channels for a total 1000 channels per team. Please reword as needed.

### CODE REVIEW

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -37,8 +37,8 @@ This article describes some of the limits, specifications, and other requirement
 |Number of members in an [org-wide team](create-an-org-wide-team.md) | 10,000       |
 |Number of teams a global admin can create        |  500,000   |
 |Number of teams a Microsoft 365 or Office 365 organization can have    | 500,000&sup3;     |
-|Number of channels per team    | 200 (includes deleted channels)<sup>4</sup>        |
-|Number of Private channels per team    |30 (includes deleted channels)<sup>4</sup>        |
+|Number of channels per team    | 1000 (includes deleted channels)<sup>4,7</sup>         |
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:40]:
```suggestion
|Number of channels per team    | 1,000 (includes deleted channels)<sup>4,7</sup>         |
```

File: Teams/limits-specifications-teams.md [hunk 2]
```diff
@@ -50,20 +50,22 @@ This article describes some of the limits, specifications, and other requirement
...
 
-<sup>4</sup> Deleted channels can be restored within 30 days. During these 30 days, a deleted channel continues to be counted towards the 200 channel or 30 private channel per team limit. After 30 days, a deleted channel and its content are permanently deleted and the channel no longer counts towards the per team limit.
+<sup>4</sup> Deleted channels can be restored within 30 days. During these 30 days, a deleted channel continues to be counted towards the 1000 channel or 30 private channel per team limit. After 30 days, a deleted channel and its content are permanently deleted and the channel no longer counts towards the per team limit.
 
 <sup>5</sup> 28 KB is an approximate limit because it includes the message itself (text, image links, etc.), @-mentions, number of connectors, and reactions.
 
 <sup>6</sup> Shared channels members from outside the team count toward this limit. Further note that teams/channel mentions are blocked in teams with over 10,000 members.
 
+<sup>7</sup> Any combination of standard and shared channels including up to 30 private channels for total of 1000 channels per team.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:59]:
```suggestion
<sup>7</sup> Any combination of standard and shared channels including up to 30 private channels for a total of 1,000 channels per team.
```

File: Teams/limits-specifications-teams.md [hunk 3]
```diff
@@ -50,20 +50,22 @@ This article describes some of the limits, specifications, and other requirement
...
+
 ### Limits for shared channels
 
 The following table describes the maximum number of channels and members.
 
 |Maximum...|Value|Notes|
 |:---------|:----|:----|
 |Members in a team|25,000|Includes all users in the team and direct members in shared channels.|
-|Shared channels per team|200|Hosted and shared with the team. (Includes deleted channels during their 30-day recovery window.)|
+|Shared channels per team|up to 1000 |Hosted and shared with the team. (Includes deleted channels during their 30-day recovery window.)|
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:68]:
```suggestion
|Shared channels per team|up to 1,000 |Hosted and shared with the team. (Includes deleted channels during their 30-day recovery window.)|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JasonLewisMSFT, kindly commit the suggested changes. Many thanks.

**Comment by JasonLewisMSFT** [comment]
Finalizing changes, committed recommended changes per JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@JasonLewisMSFT, thank you for committing my suggestions, but I noticed that you have now closed the PR. Was that on purpose?

**Comment by JasonLewisMSFT** [comment]
> @JasonLewisMSFT, thank you for committing my suggestions, but I noticed that you have now closed the PR. Was that on purpose?

I thought I committed the changes.  Re-opening the PR now.

**Comment by JasonLewisMSFT** [comment]
@JohanFreelancer9 I believe it is committed now?  If not let me know next steps.

**Comment by scanum** [comment]
@MicrosoftHeidi 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JasonLewisMSFT** [comment]
When will the changes go live?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
> When will the changes go live?

@scanum

**Comment by scanum** [comment]
@MicrosoftHeidi
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**